src/server/data/validate.js

```diff
@@ -1,12 +1,25 @@
+const fs = require('fs');
 const Ajv = require('Ajv');
 const schema = require('./schema.json');
-const data = require('./learners/SonyaSims.json');
 const ajv = new Ajv();
 const validate = ajv.compile(schema);
-const valid = validate(data);
 
-if (!valid) {
-  console.log(validate.errors);
-} else {
-  console.log('Format is correct');
-}
+const checkLearners = () => {
+  fs.readdir('./learners', 'utf8', (error, files) => {
+    if (error) {
+      throw error;
+    }
+
+    files.forEach((file) => {
+      let data = require(`./learners/${file}`);
+      const valid = validate(data);
+      if (!valid) {
+        console.log(validate.errors);
+      } else {
+        console.log('Format is correct');
+      }
+    });
+  });
+};
+
+checkLearners();
```

Comment:
hmmmm... i think this was branched from another branch, and not from master, but the end result should be the same =O i'll keep an eye on it.

Reply:
nm, it's a lot of commits -- asking for rebase